src/main/java/tr/edu/ogu/ceng/controller/StudentController.java

```diff
@@ -1,5 +1,24 @@
 package tr.edu.ogu.ceng.controller;
 
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import lombok.AllArgsConstructor;
+import tr.edu.ogu.ceng.dto.requests.UpdateStudentRequest;
+import tr.edu.ogu.ceng.dto.responses.UpdateStudentResponse;
+import tr.edu.ogu.ceng.service.StudentService;
+
+@RestController
+@RequestMapping("api/students") 
+@AllArgsConstructor
 public class StudentController {
 
+	private StudentService studentService;
+	
+	 @PutMapping("/update")
+	public UpdateStudentResponse update (long id, @RequestBody  UpdateStudentRequest updateStudentRequest)
```

Comment:
neden id ayrı bir parametre olarak geliyor. 

+ Her request için ayrı yapı konusunu eğer net bir şekilde eminseniz sizin grubun diğer gruplarla paylaşması gerekiyor ki herkes aynı yapıyı benimsesin.